packages/ui-heading/src/Heading/index.js

```diff
@@ -130,16 +126,10 @@ class Heading extends Component {
     return (
       <View
         {...passthroughProps(props)}
-        className={classnames({
-          [styles.root]: true,
-          [styles[`level--${level}`]]: true,
-          [styles[`color--${color}`]]: color,
-          [styles[`border--${border}`]]: border !== 'none',
-          [styles.ellipsis]: ellipsis
-        })}
+        css={this.props.styles.heading}
         as={ElementType}
-        margin={margin}
         elementRef={elementRef}
+        margin={this.props.styles?.forwardedStyleProps?.margin}
```

Comment:
I don't think we need to do this since the View component will override the margin with the inline applied styles.

Reply:
Removed